src/rt/dmain2.d

```diff
@@ -223,7 +223,7 @@ extern (C) int rt_init()
  */
 extern (C) int rt_term()
 {
-    if (!_initCount) return 0; // was never initialized
+    if (atomicOp!"!="(_initCount,0)) return 0; // was never initialized
```

Comment:
```suggestion
    if (atomicOp!"=="(_initCount, 0)) return 0; // was never initialized
```

The diff as is turns `_initCount == 0` into `_initCount != 0`. Also, style fix.

Reply:
Ahh, true.